src/routes/+page.svelte

```diff
@@ -11,20 +11,20 @@
 
 	const languages = [
 		{ value: 'english', label: 'English' },
-		{ value: 'france', label: 'France' },
-		{ value: 'italian', label: 'Italian' },
+		{ value: 'france', label: 'Français (French)' },
```

Comment:
I would make more sense to rename france → french.

```suggestion
		{ value: 'french', label: 'Français (French)' },
```

It would require to modify the key `france` in:

https://github.com/codenoid/github-roast/blob/cc482bb10abc14edc4ef6ca1663dfbca9017ea27/src/routes/llama/%2Bserver.js#L22
https://github.com/codenoid/github-roast/blob/cc482bb10abc14edc4ef6ca1663dfbca9017ea27/src/routes/llama/%2Bserver.js#L150-L152

Reply:
Gotchya! I'll make the changes!